### ISSUE TITLE
Using Patroni at Disaster Recovery Site

### ISSUE THREAD

**Issue by heshammostafa07** [zalando/patroni#801]
Hello,
I have 2 sites Site A (Primary site) and Site B (Disaster recovery site) . I am using Patroni at site A installed at  2 servers and working fine. I need to use Patroni to let server at site B as leader when both servers at site A stopped (total failure at site A).

**Comment by CyberDem0n** [comment]
Hi,

I think you are looking for something like https://github.com/zalando/patroni/pull/679

**Comment by heshammostafa07** [comment]
Hello,
Exactly this will fulfill my needs. But it seams that it is not completed yet

**Comment by ik234** [comment]
will this work with only 1DB server on each site? May be someone have a clue or some other solution?

**Comment by CyberDem0n** [comment]
> will this work with only 1DB server on each site?

Well, it should. But in your case it would be useless. https://github.com/zalando/patroni/pull/679 won't do automatic failover to the second site, you'll have to do it manually.

Basically you can get absolutely the same behavior by starting two nodes in two data-centers and setting up streaming replication.

**Comment by CyberDem0n** [comment]
> But it seams that it is not completed yet

You are right, it is not 100% ready, but it is very close to this state and you can help us to test it.

**Comment by heshammostafa07** [comment]
Sure I will once it is ready
